### ISSUE TITLE
Broken layout navigation bar

### ISSUE THREAD

**Issue by AntoineChwat** [52inc/Pulley#247]
Hi, I've been working with pulley for a couple days and I need to work with older devices running iOS 9. So I tried your project on a simulator with iOS 9.3 and I get the following layout:
![simulator screen shot - iphone 6 - 2018-08-17 at 11 30 25](https://user-images.githubusercontent.com/15050612/44259233-40cb2900-a211-11e8-9955-f504f5213f83.png)
![simulator screen shot - iphone 6 - 2018-08-17 at 11 30 32](https://user-images.githubusercontent.com/15050612/44259243-46c10a00-a211-11e8-930b-3a5088fc383b.png)
The screenshots are from my own application but you can get the exact same results if you embed the PulleyViewController from the sample application inside a Navigation Controller and then run the project on a simulator running iOS 9.3 or iOS 10.3.1, iOS 11.1 and 11.4 work flawlessly though. No issues if the nav bar is hidden either. Any idea on how to fix this?

Cheers!

**Comment by brendan09** [comment]
Although a bug, this is likely due to the topLayoutGuide a translucent navigation bar is providing.

Try setting the navigation bar’s translucency to false and see if that helps.

**Comment by AntoineChwat** [comment]
I added `self.pulleyViewController?.navigationController?.navigationBar.isTranslucent = false` in the drawer's `viewWillAppear` in the sample project and this is what I get:
![simulator screen shot - iphone 6 - 2018-08-17 at 17 15 49](https://user-images.githubusercontent.com/15050612/44274026-6839ea00-a241-11e8-9820-61b1d79bc64d.png)
![simulator screen shot - iphone 6 - 2018-08-17 at 17 15 24](https://user-images.githubusercontent.com/15050612/44274032-6a9c4400-a241-11e8-9123-e2994ce6b8db.png)
Basically, I can see the bottom of the drawer when it is open and it doesn't collapse any further than this, maybe I can do something about it programatically? I'll try to look into it next week
